scripts/superdesk-authoring/authoring.js

```diff
@@ -2352,6 +2352,7 @@
                         })
                         .then(function(value) {
                             notify.success(gettext('Item sent.'));
+                            $rootScope.$broadcast('item:fetch');
```

Comment:
I'm not 100% familiar with the functionality here, but wouldn't `item:move` be more appropriate? It seems like the item is being sent / moved, not fetched. `item:move` is a new event that has been added by the backend and there could already be handlers in place.